Doc/whatsnew/3.8.rst

```diff
@@ -234,6 +234,15 @@ Added method :meth:`~tkinter.Canvas.moveto`
 in the :class:`tkinter.Canvas` class.
 (Contributed by Juliette Monsel in :issue:`23831`.)
 
+time
+----
+
+Added new clock :data:`~time.CLOCK_UPTIME_RAW` for macOS 10.12 that 
+increments monotonically, tracking the time since an arbitrary point, 
+unaffected by frequency or time adjustments and not incremented while the 
+system is asleep.
```

Comment:
I would prefer just "Added new clock :data:`~time.CLOCK_UPTIME_RAW` for macOS 10.12." Since the doc generates a link to the doc, there is no need to repeat the constant documentation.